### PR TITLE
DOC-1984: Added TINY-9698 to the release notes.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-1984

Changes:
* Added TINY-9698 to the release notes.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added
- [x] ~`modules/ROOT/nav.adoc` has been updated (if applicable)~
- [x] ~Files has been included where required (if applicable)~
- [x] ~Files removed have been deleted, not just excluded from the build (if applicable)~
- [x] ~(New product features only) Release Note added~

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 1]
```diff
@@ -30,6 +30,21 @@
...
+=== Advanced Code Editor 3.3.0
+
+The {productname} 6.5 release includes an accompanying release of the **Advanced Code Editor** premium plugin.
+
+**Advanced Code Editor** 3.3.0 includes the following fix.
+
+=== The `copy code` button wasn't correctly styled within the Advanced Code Editor.
+//#TINY-9698
+
+In previous versions of {productname}, the "Copy code" button had a grey background.
```
Comment (lorenzo-pomili) [inline modules/ROOT/pages/6.5-release-notes.adoc:42]: there were no previous implementation of this button, is just that the first implementation was wrongly styled

**Review by lorenzo-pomili — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 2]
```diff
@@ -30,6 +30,21 @@
...
+
+This issue was caused by the previous setting of `borderless: true` which should have been set to `false`. As a consequence, the button wasn't correctly styled.
```
Comment (kemister85) [inline modules/ROOT/pages/6.5-release-notes.adoc:44]:
```suggestion
In {productname}, implementation for the new "Copy code" button was styled incorrectly with a grey background.

This issue was a oversight which meant the setting for `borderless: true` should have been set to `false`. As a consequence, the button wasn't correctly styled. 
```

Comment (kemister85) [reply modules/ROOT/pages/6.5-release-notes.adoc:44]: @lorenzo-pomili Please review suggested changes, and remove blocker from this if you are happy with the changes.

File: modules/ROOT/pages/6.5-release-notes.adoc (hunk 1 above)
Comment (kemister85) [reply modules/ROOT/pages/6.5-release-notes.adoc:42]: @lorenzo-pomili please see below suggestion

File: modules/ROOT/pages/6.5-release-notes.adoc (hunk 2 above)
Comment (lorenzo-pomili) [reply modules/ROOT/pages/6.5-release-notes.adoc:44]: @kemister85 the incorrect style was the one without the grey background

**Review by lorenzo-pomili — CHANGES_REQUESTED** [review]

**Review by lorenzo-pomili — APPROVED** [review]

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 3]
```diff
@@ -30,6 +30,21 @@
...
+
+=== The `copy code` button wasn't correctly styled within the Advanced Code Editor.
+//#TINY-9698
+
+In {productname}, implementation for the new "Copy code" button was styled incorrectly which meant it was missing the grey background similar to the other buttons.
+
+This issue was a oversight which meant the setting for `borderless: true` should have been set to `false`. As a consequence, the button wasn't correctly styled. 
+
+In {productname} 6.5, the `borderless: true` setting has now been removed and the button is now correctly styled.
+
```
Comment (brian-forte) [inline modules/ROOT/pages/6.5-release-notes.adoc:47]:
```suggestion
=== Advanced Code Editor 3.3.0

The {productname} 6.5 release includes an accompanying release of the **Advanced Code Editor** premium plugin.

**Advanced Code Editor** 3.3.0 includes the following fix.

=== The _Copy code_ button was incorrectly styled within the Advanced Code Editor.
//#TINY-9698

In previous versions of {productname}, the _Copy code_ button was styled incorrectly. Specifically, it was missing the grey background used in other, equivalent, buttons.

This was caused by an incorrect CSS setting. The setting, `borderless: true`, should not have been applied. As a consequence, the button wasn't correctly styled. 

For {productname} 6.5, the `borderless: true` setting has been removed and the button is now correctly styled.

```

Edits and copy-edits.

**Review by brian-forte — APPROVED** [review]

**Pull request merged**